### ISSUE TITLE
Entire Israel West Bank are is not recognized as Israel

### ISSUE THREAD

**Issue by eladraviv** [dewi-alliance/hplans#30]
### Hotspot b58 Addresses

112J8ZQkwHHC6neA6LtTVyTkqgaEQkBwCpd1idoJUhW57Y3HuQGc
1124qLC1hrC47yU2RFqTT4NHHYCEaEJujkeN3hH3RTaj7K23fGJx
119QLi6AZuhizziKGG4WEECmXyCYHjwsaRnT2Z1zxvws3RoJF87

### Discord Handle

el69#6601

### What regional channel plan do you expect your hotspot to be using?

AS923-4

### Details

Hotpots in Israel in the West Bank have a major issue that their country returns null from the Helium API:
https://api.helium.io/v1/hotspots/112J8ZQkwHHC6neA6LtTVyTkqgaEQkBwCpd1idoJUhW57Y3HuQGc

The value is null probably because the West Bank area is not recognized to be in Israel.
Because country is null, miners cannot assign frequency and can't participate in Proof of Coverage at all.

**Potential reason**
Since this is the West Bank maybe some political reasons are in play and the location is not identified as Israel, but for all things matter at this point this is part of Israel and should be recognize so for any miner to be able to work there.

**Hotspots example**
Note that there is no country:
![image](https://user-images.githubusercontent.com/32803952/160418954-a5a6f746-bf58-445a-b963-5ac307e4fd68.png)

Example hotspot with country (not in the West Bank):
![image](https://user-images.githubusercontent.com/32803952/160419067-cdea31de-5f3e-4e94-aec9-39b0da1bc1aa.png)

**Problematic area**
As described all of the hotspots in the west bank are not being allocated to Israel as they supposed to.
Higher view, in red - Israel border line, In blue - the area which Helium not recognize to be in Israel:
![image](https://user-images.githubusercontent.com/32803952/160419247-c8646290-198b-4801-9380-e9538cb9c6fb.png)
Any hotspot inside the blue line are not allocated to Israel, even though this is the center of Israel.

**Impact**
Helium is growing rapidly in the last couple months in Israel and there are many cities in the west bank (no matter the ethnic relation) that will not be able to participate as any hotspot won't have any country allocation.

**Testing**
I placed 3 hotspots in this area and asserted the location - none of them could identify the region or participating in PoCs for 2 weeks.
After 2 weeks of no activity I took 2 of the hotspots and asserted them to a new location very close (but this time not on the west bank area), without changing any actual configuration or setup, and right after 1 hour the region appeared as Israel and they started to witness beacons and being challenged.

**Some more examples**
https://explorer.helium.com/hotspots/hex/882db07329fffff
https://explorer.helium.com/hotspots/hex/882db0701bfffff
https://explorer.helium.com/hotspots/hex/882db00837fffff
Curved Ebony Cobra Sensecap dashboard:
![image](https://user-images.githubusercontent.com/32803952/160419843-9e432097-56a1-49d8-bd6d-bf1cfeda2ba1.png)

How can we solve this problem for the benefits of everyone?

Thanks,
Elad.

**Comment by HNT-Hacker** [comment]
https://undocs.org/en/a/res/58/292
Free Palestine

**Comment by eladraviv** [comment]
@HNT-Hacker I don't think this is the place for politics. Just note that also Palestinians hotspots are not working due to the issue above.
The bug dose not discriminate by ethnic origin.

**Comment by eladraviv** [comment]
Any chance someone can check this issue?

Thanks

**Comment by gradoj** [comment]
![image](https://user-images.githubusercontent.com/7544765/165778528-7b9ee3a8-20cb-497d-8d74-cc466892f72f.png)

i agree with the above and suggest we make Israel one region all AS923-4. currently the white regions are set as unknown and dark blue as923-4
